pkg/compose/build_classic.go

```diff
@@ -252,25 +211,18 @@ func isLocalDir(c string) bool {
 	return err == nil
 }
 
-func imageBuildOptions(options buildx.Options) dockertypes.ImageBuildOptions {
+func imageBuildOptions(config *types.BuildConfig) dockertypes.ImageBuildOptions {
```

Comment:
Perhaps rename `classicBuildOptions()`

Reply:
this produces the `options` for the `ImageBuild` engine API, so ...